packages/xrpl/test/client/autofill.test.ts

```diff
@@ -23,6 +24,8 @@ const HOOKS_TESTNET_ID = 21338
 
 describe('client.autofill', function () {
   let testContext: XrplTestContext
+  let paytxn
+  let amount_: string
```

Comment:
```suggestion
  let paymentTx
  const AMOUNT = '1234'
```

Reply:
updated in [28d1ef4](https://github.com/XRPLF/xrpl.js/pull/2689/commits/28d1ef47d738527dc4d4774c81629f0bae8dc521)